README.md

```diff
@@ -25,10 +25,10 @@ pip install git+https://github.com/huggingface/optimum-intel.git
 ```
 
 To install the latest release of this package with the corresponding required dependencies, you can do respectively:
-| Accelerator | Installation |
-|:------------|:---------------------------------------------|
-| [OpenVINO](https://docs.openvino.ai/latest/index.html)| `python -m pip install optimum-intel[openvino]` |
-| Intel [Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) | `python -m pip install optimum-intel[neural-compressor]`|
+| Accelerator                                                                                                      | Installation                                      |
+|:-----------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
+| [OpenVINO](https://docs.openvino.ai/latest/index.html)                                                           | `python -m pip install optimum[openvino]`         |
```

Comment:
How about NNCF `python -m pip install optimum-intel[nncf]`?

Reply:
I have added it !